near/omni-bridge/src/lib.rs

```diff
@@ -981,6 +960,29 @@ impl Contract {
         )
     }
 
+    #[payable]
+    #[access_control_any(roles(Role::DAO))]
+    pub fn set_utxo_chain_token(
```

Comment:
Lets combine it with `add_utxo_chain_connector`, this will simplify configuring new chains

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/b551c41bfeb67d41ea8477ebf7f676f75636fd8d